torchao/utils.py

```diff
@@ -65,3 +66,18 @@ def find_multiple(n: int, *args: Tuple[int]) -> int:
     if n % k == 0:
         return n
     return n + k - (n % k)
+
+if version.parse(torch.__version__) >= version.parse("2.4.0.dev"):
```

Comment:
Thanks for this! I wish for this PR we had seperated out the easy utils changes from the more controversial BC changes. Would make this easier to merge

Reply:
OK sure

Reply:
https://github.com/pytorch/ao/pull/337